src/components/Book.js

```diff
@@ -1,15 +1,31 @@
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/bookSlice';
+
 const Book = (book) => {
+  const dispatch = useDispatch();
   const {
-    type, title, author, chapter,
+    type, title, author, chapter, id,
   } = book;
+  const handleDelete = (id) => {
+    dispatch(removeBook({ id }));
+  };
   return (
     <div className="theBook">
       <h3 className="bookType">{type}</h3>
       <h1 className="bookTitle">{title}</h1>
       <h2 className="bookAuthor">{author}</h2>
       <ul className="bookOptions">
         <li className="option">Comments</li>
-        <li className="option">Remove</li>
+        <li>
+          <button
+            className="option"
+            onClick={() => { handleDelete(id); }}
+            onKeyDown={() => { handleDelete(id); }}
+            type="button"
+          >
+            Remove
```

Comment:
- [ ] When I try to remove a book, all books get removed, that's not supposed to happen, Please fix that.

![chrome-capture-2023-2-9](https://user-images.githubusercontent.com/68796232/223880218-9911e8b7-d78d-45cc-8f61-dfea3bafcc41.gif)